ci/jenkins/mellanox/jobs/00-functions.yaml

```diff
@@ -0,0 +1,133 @@
+- builder:
+    name: run-ci
+    builders:
+         - shell:
+            !include-raw: ../common/run_ci_scripts.sh
+
+- builder:
+    name: get-ci-scripts
+    builders:
+         - shell: |
+            #!/bin/bash -x
+            rm -rf $WORKSPACE/*
+            scripts_dir='ci/jenkins/mellanox'
+            if [[ ! -d /jenkins/workspace/${UPSTREAM_JOB_NAME}/PR/${PR_ID}/${scripts_dir} ]];then
+                git clone $ANTREA_REPO $WORKSPACE/antrea
+                mv $WORKSPACE/antrea/${scripts_dir}/* $WORKSPACE/
+            else
+                mv /jenkins/workspace/${UPSTREAM_JOB_NAME}/PR/${PR_ID}/${scripts_dir}/* $WORKSPACE/
+            fi
+
+- builder:
+    name: builder-pending-label
+    builders:
+      - shell: 'exit 1 # fail on purpose'
+
+- builder:
+    name: post-pending-page
+    builders:
+      - shell: |
+          ssh $LOGSERVER mkdir -p /var/www/html/${JOB_NAME}
+          ssh $LOGSERVER mkdir -p /var/www/html/${JOB_NAME}/${BUILD_ID}
+          scp /jenkins/pending_page.html $LOGSERVER:/var/www/html/${JOB_NAME}/${BUILD_ID}/consoleFull
+
+- builder:
+    name: post-in-progress-page
+    builders:
+      - shell: |
+          scp /jenkins/in_progress.html $LOGSERVER:/var/www/html/${UPSTREAM_JOB_NAME}/${UPSTREAM_BUILD_ID}/consoleFull
+
+- builder:
+    name: post-full-console
+    builders:
+      - shell: |
+          #!/bin/bash +x
+          set -x
+          sleep 10
+
+          env
+
+          # note(abdallahyas): we need to check if the downstream job was finished
+          # normally (either succeeded or failed) within the timeout period. This
+          # can be done by checking if the LAST_TRIGGERED_JOB_NAME is  passed to the
+          # upstream job, if not that means either the downstream job was aborted for
```

Comment:
VMware recommends to stop using "abort" / "kill" as part of its inclusive naming initiative (https://inclusivenaming.org/). Please consider an alternative, like "stop".

Reply:
Done. Used stop.